[2023-09-29T09:53:43.189+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.API_readiness_check_call manual__2023-09-29T09:53:38.746957+00:00 [queued]>
[2023-09-29T09:53:43.194+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.API_readiness_check_call manual__2023-09-29T09:53:38.746957+00:00 [queued]>
[2023-09-29T09:53:43.194+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-09-29T09:53:43.207+0000] {taskinstance.py:1380} INFO - Executing <Task(HttpSensor): API_readiness_check_call> on 2023-09-29 09:53:38.746957+00:00
[2023-09-29T09:53:43.212+0000] {standard_task_runner.py:57} INFO - Started process 3566 to run task
[2023-09-29T09:53:43.220+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'weather_dag', 'API_readiness_check_call', 'manual__2023-09-29T09:53:38.746957+00:00', '--job-id', '67', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmp6di203x3']
[2023-09-29T09:53:43.221+0000] {standard_task_runner.py:85} INFO - Job 67: Subtask API_readiness_check_call
[2023-09-29T09:53:43.263+0000] {task_command.py:415} INFO - Running <TaskInstance: weather_dag.API_readiness_check_call manual__2023-09-29T09:53:38.746957+00:00 [running]> on host ip-172-31-24-192.eu-north-1.compute.internal
[2023-09-29T09:53:43.324+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='API_readiness_check_call' AIRFLOW_CTX_EXECUTION_DATE='2023-09-29T09:53:38.746957+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-29T09:53:38.746957+00:00'
[2023-09-29T09:53:43.325+0000] {http.py:130} INFO - Poking: /data/2.5/weather?lat=40.6403167&lon=22.9352716&appid=35d0d7526bceb84756b29a9ece8b28f3
[2023-09-29T09:53:43.330+0000] {base.py:73} INFO - Using connection ID 'weathermap_api' for task execution.
[2023-09-29T09:53:43.567+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-09-29T09:53:43.573+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=API_readiness_check_call, execution_date=20230929T095338, start_date=20230929T095343, end_date=20230929T095343
[2023-09-29T09:53:43.629+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-29T09:53:43.656+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
